### ISSUE TITLE
[BUG]: disabled bl-select can be updated

### ISSUE THREAD

**Issue by AykutSarac** [Trendyol/baklava#793]
### Issue description

Despite bl-select being disabled, if you click onto it and focus then press space it will open the popover and allow you to edit your selection.

### Media & Screenshots

![image](https://github.com/Trendyol/baklava/assets/47941171/d2d8ddd3-e376-4a1b-b76e-c4ab7f71c0b9)

### Priority this issue should have

High (immediate attention needed)

### Please review the checkboxes that are applicable.

- [ ] This issue is directly related to React
